### ISSUE TITLE
Adding new currency

### ISSUE THREAD

**Issue by mrkurosawa** [peatio/peatio#663]
Do all by guide https://gist.github.com/brossi/175f60bd1dc4f99f9373
But old users doesnot have new account, only new users. What i should do for add accounts for ALL users?
funds_controller.rb
```
def index
      abort( "#{current_user.accounts.length}" )
```
Old users show 2, new users 3.

**Comment by mrkurosawa** [comment]
I can manually create record in accounts table but is there command to generate accounts for all users?

**Comment by legends420** [comment]
i created a script to add the new currency to all old users

**Comment by mrkurosawa** [comment]
Yes i could but i am pretty new in ruby. I am php coder, i though that there is a easy way to do it with peatio. Can you share your script?

**Comment by legends420** [comment]
i coded it in PHP i would share for a small donation for my time join my discord https://discord.gg/fNqTj2F

**Comment by Apollo-IT** [comment]
I've already developed and deployed 3 exchanges based on Peatio.
I can add more coins and customize exchanges with peatio.
Also have experiences in wlox.
Feel free to message me any time.
**skype : live:apollo447722
mail : [apollo447722@outlook.com](mailto:apollo447722@outlook.com)**
Regards.
